### PR TITLE
[ENHO]New field Customizing

### PR DESCRIPTION
**Pull request by huber-nicolas**

### CODE REVIEW

File: file-formats/enho/type/zif_aff_enho_v1.intf.abap [hunk 1]
```diff
@@ -13,6 +13,26 @@ INTERFACE zif_aff_enho_v1
...
+      "! <p class="shorttext">Customizing supported</p>
+      "! Customizing is supported
+      customizing_supported              TYPE ty_customizing VALUE ' ',
+      "! <p class="shorttext">Customizing not supported</p>
+      "! Customizing is not supported
+      customizing_not_supported              TYPE ty_customizing VALUE 'X',
+      "! <p class="shorttext">Customizing supported (no transport)</p>
+      "! Customizing is supported (without transport)
+      customizing_supported_no_trans              TYPE ty_customizing VALUE 'D',
+      END OF co_customizing.
```
Comment (schneidermic0) [inline file-formats/enho/type/zif_aff_enho_v1.intf.abap:34]: Shall we just remove the prefix `customizing_`. If we remove it we get also a better text for the scenarios without transport
```suggestion
      supported              TYPE ty_customizing VALUE ' ',
      "! <p class="shorttext">Customizing not supported</p>
      "! Customizing is not supported
      not_supported              TYPE ty_customizing VALUE 'X',
      "! <p class="shorttext">Customizing supported (no transport)</p>
      "! Customizing is supported (without transport)
      supported_no_transport              TYPE ty_customizing VALUE 'D',
    END OF co_customizing.
```

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
Thanks. See my comments below

File: file-formats/enho/type/zif_aff_enho_v1.intf.abap (hunk 1 above)
Comment (huber-nicolas) [reply file-formats/enho/type/zif_aff_enho_v1.intf.abap:34]: yes, I will adapt it in UIA

**Review by schneidermic0 — APPROVED** [review]
Looks good to me. Thanks :)

**Pull request merged**